client/post-editor/media-modal/index.jsx

```diff
@@ -209,15 +209,15 @@ module.exports = React.createClass( {
 		this.setView( ModalViews.IMAGE_EDITOR );
 	},
 
-	onImageEditorClose: function() {
+	onImageEditorClose: function( event, { hasEditedImage } = { hasEditedImage: false } ) {
```

Comment:
>I pass this in the onDone handler,

Ah, thought I did, but I didn't have the branch checked out locally when looking at the file.